Source/WebInspectorUI/UserInterface/Views/CookiePopover.js

```diff
@@ -245,6 +257,18 @@ WI.CookiePopover = class CookiePopover extends WI.Popover
 
         toggleExpiresRow();
 
+        let togglePartitionKeyRow = () => {
+            partitionKeyInputRow.rowElement.hidden = !this._partitionedCheckboxElement.checked;
```

Comment:
we should probably also only show the `partitionKey` if it's set
```js
partitionKeyInputRow.rowElement.hidden = !this._partitionedCheckboxElement.checked || !data.partitionKey;
```
alternatively we could not even make the `partitionKeyInputRow` if the `partitionKey` is not set (e.g. when we're creating an entirely new cookie from scratch)